### PR TITLE
[PHP] Add Mark

### PR DESCRIPTION
**Pull request by passwalls**
Add Mark.

Mark is a high performance micro framework based on workerman helps you quickly write simple APIs with php.

Thank you for your project 👍

### CODE REVIEW

File: php/mark/composer.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "require": {
+        "mark-php/mark": "^1.0"
```
Comment (waghanza) [inline php/mark/composer.json:3]: could you please use `~` ? -> `~1.0.0`

Comment (passwalls) [reply php/mark/composer.json]: Ok. Changed.

**Comment by waghanza** [comment]
@passwalls May I also add you to `CODEOWNERS` ? so as to ping you on each modification of `mark` ?

**Comment by passwalls** [comment]
Yes, of course. Thank you.

**Comment by waghanza** [comment]
@passwalls Results updated => https://github.com/the-benchmarker/web-frameworks#results :tada:

**Pull request merged**